scripts/release-hotfix.js

```diff
@@ -0,0 +1,30 @@
+const currentVersion = require('../.github/actions/helpers/current-version');
+const {execSync} = require('child_process');
+const semver = require('semver');
+const hotfixBranchName = 'tmp_hotfix_branch';
```

Comment:
And now here the `tmp_hotfix_branch` comes into the game. As said above, if we go for the major.minor branches, IMO this is not necessary.